src/modules/api/pipeline-task/routes/task.py

```diff
@@ -128,4 +122,4 @@ def update_task():
   # If the job has finished or errored out, acknowledge the Pub/Sub message
   # If the job is still running, don't acknowledge -- tells Pub/Sub to try the request again
   # Return as bool value to be handled by pubsub_endpoint decorator
-  return op['done'] or op['error']
+  return status in JobStatus.FINISHED
```

Comment:
Since the `PipelineOperation` is now managed entirely by `GCPRunner`, we instead compare the resulting status to the set of "finished" status values (which in my opinion is a better solution anyways).